astropy/cosmology/io/row.py

```diff
@@ -26,14 +26,17 @@ def from_row(row, *, move_to_meta=False, cosmology=None):
         (e.g. for ``Cosmology(meta={'key':10}, key=42)``, the ``Cosmology.meta``
         will be ``{'key': 10}``).
 
-    cosmology : str, `~astropy.cosmology.Cosmology` class, or None (optional, keyword-only)
+    cosmology : str, |Cosmology| class, or None (optional, keyword-only)
```

Comment:
Shouldn't this be written as
```suggestion
    cosmology : str, type, or None (optional, keyword-only)
```